src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -66,6 +66,21 @@ public class AzureBlobCache : ImageCacheBase
         private string cachedRewritePath;
 
         /// <summary>
+        /// gets reference or creates container
+        /// </summary>
+        /// <param name="cloudBlobClient">cloud blob client</param>
+        /// <param name="containerName">name of container</param>
+        /// <param name="accessType">access permissions ie. public</param>
+        /// <returns>container for cache items</returns>
+        private CloudBlobContainer CreateContainer(CloudBlobClient cloudBlobClient, string containerName, BlobContainerPublicAccessType accessType)
```

Comment:
This is a great addition! I didn't know you could do that.

However, putting my StyleCop hat on :cop:, private methods should always be below public ones and I specifically avoid using var for anything other than anonymous types in this project.